mne/io/meas_info.py

```diff
@@ -283,7 +283,7 @@ class Info(dict, MontageMixin):
     gantry_angle : float | None
         Tilt angle of the gantry in degrees.
     lowpass : float
-        Lowpass corner frequency in Hertz.
+        Lowpass corner frequency in Hertz. It is automatically set to half the sampling rate if there is otherwise no low-pass applied to the data.
```

Comment:
this will yield a "line too long" error from our style checker.  Max line length is 79 characters.

Your IDE should have a "rulers" setting that will add a vertical line after 79 characters, this can help you notice such problems before pushing.  Also running `make flake` from the root of the local MNE-Python clone would have caught this.